RELEASE/scripts/sl_ascend.ash

```diff
@@ -14665,46 +14729,58 @@ boolean doTasks()
 		}
 	}
 
-	if((get_property("hippiesDefeated").to_int() < 64) && (get_property("fratboysDefeated").to_int() < 64) && (my_level() >= 12) && (get_property("sl_prewar") == "started") && (get_property("sl_war") != "finished"))
+	if(in_koe())
 	{
-		print("First 64 combats. To orchard/lighthouse", "blue");
-		if((item_amount($item[Stuffing Fluffer]) == 0) && (item_amount($item[Cashew]) >= 3))
+		if((get_property("hippiedDefeated").to_int() < 300) && (get_property("fratboysDefeated").to_int() < 300))
```

Comment:
Isn't it supposed to be 333 instead of 300? My understanding is that it SAYS 300 mistakenly, but 333 is the actual functional and intended number.

Reply:
You're quite right. The Battlefield is still really janky so I didn't get to see this - I just ran those turns manually.

Reply:
```suggestion
		if((get_property("hippiedDefeated").to_int() < 333) && (get_property("fratboysDefeated").to_int() < 333))
```